datafusion/core/Cargo.toml

```diff
@@ -47,7 +47,6 @@ parquet = ["datafusion-common/parquet", "dep:parquet"]
 pyarrow = ["datafusion-common/pyarrow", "parquet"]
 regex_expressions = ["datafusion-physical-expr/regex_expressions", "datafusion-optimizer/regex_expressions"]
 serde = ["arrow-schema/serde"]
-simd = ["arrow/simd"]
```

Comment:
Can we also remove the `simd` feature from the main readme as well?

https://github.com/apache/arrow-datafusion/blob/746988a7e5c9f256c3b24ab7e3f30ffd90d542a0/README.md?plain=1#L72

Reply:
Done in 727b88144